yaksh/file_utils.py

```diff
@@ -0,0 +1,33 @@
+import shutil
```

Comment:
file_utils is standard name which  resembles file related features. So I feel this is appropriate since here we are only file doing file related functionalities.